data/mods/Magiclysm/npc/TALK_FORGE_DIVINER.json

```diff
@@ -508,15 +515,15 @@
     ]
   },
   {
-    "id": "TALK_FORGE_LORD_DIVINER_ORC",
+    "id": "TALK_FORGE_LORD_DIVINER_GOBLIN_OUTPOST",
     "type": "talk_topic",
-    "dynamic_line": "Militaristic xenophobes, that bunch.  They are fairly good weapon smiths, and quite strong, so be careful.  Oh, and something about their physiology makes them extra super dangerous when raised as undead.  So you should probably try to not let that happen.  I'll mark one of their camps on your map for one denarius.",
+    "dynamic_line": "Contrary to popular belief, goblins are not actually orcs.  Its the other way around.  Orcs are goblins who have taken one too many unlabeled potions and lived to tell the tale.  Sometimes this awakens powers much like my own.  Well, not exactly, if it did I'd be out of a job faster than a Magus could cast Phase Door.  Luckily for you, you shouldn't have to delve too deep into a forest to see for yourself. I'll mark one of their outposts on your map for one denarius.",
```

Comment:
This feels off compared to the other additions. Maybe a bit too educational?

Reply:
The reason I went with this more educational spiel was trying to have ut be more in line with a few of the other dialogue she has when giving monster related missions, with a little tip on what to expect. However I might have gone a little bit overboard especially in the beginning, but I wanted to get a few of those little flavor regarding Orcs that the author of the giblin/orc overhaul desired into the game